pkg/routeagent/controllers/route/route.go

```diff
@@ -359,7 +359,7 @@ func (r *Controller) createVxLANInterface(ifaceType int, gatewayNodeIP net.IP) e
 		}
 	}
 
-	err = r.vxlanDevice.configureIPAddress(vtepIP, vtepMask.Mask)
+	err = r.vxlanDevice.configureIPAddress(vtepIP, net.CIDRMask(8, 32))
```

Comment:
Technically we could even go down to /4 if we wanted.

Reply:
True. If required, we can do that.